spec/services/status_update_service_spec.rb

```diff
@@ -26,18 +26,13 @@
                             signon_user_uid: user.uid)
     end
 
-    it "updates the email status" do
-      expect { described_class.call(args) }
-        .to(change { email.reload.status })
-    end
-
     context "when provided a 'delivered' status" do
       let(:status) { "delivered" }
 
       it "sets the delivery attempt status to delivered" do
         expect { described_class.call(args) }
-          .to change { delivery_attempt.reload.status }
-          .to("delivered")
+          .to change { delivery_attempt.reload.status }.to("delivered")
+          .and change { email.reload.finished_sending_at }.to(sent_at)
```

Comment:
Minor: the use of `sent_at` is (now) confusing here, and seems redundant since we also have this `completed_at` variable, which is set to the same value. Can we just say `timestamp` or `status_update_time` (or something)?